R-packages/evalcast/DESCRIPTION

```diff
@@ -26,7 +35,10 @@ Authors@R:
     person(given = "Ryan",
            family = "Tibshirani",
            role = "aut"))
-Description: Evaluation tools for probabilistic COVID-19 forecasters. This
+URL: https://cmu-delphi.github.io/covidcast/evalcastR/,
```

Comment:
At the moment, this link leads to a 404, but I think @dajmcdon is updating the docs?

Reply:
I updated pkg docs in #248 on the `evalcast` branch. I haven't done a PR against main yet to incorporate, but we can...